ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -12,18 +12,26 @@
 
   {{#each @destination.formFieldGroups as |fieldGroup|}}
     {{#each-in fieldGroup as |group fields|}}
-      {{#if (and (eq group "Credentials") (not @destination.isNew))}}
+      {{! h2 titles Credentials and Configuration for FormFieldGroups to be displayed in both create and edit }}
+      {{#if (eq group "Credentials")}}
         <hr class="has-top-margin-xl has-bottom-margin-l has-background-gray-200" />
         <Hds::Text::Display @tag="h2" @size="400" @weight="bold">Credentials</Hds::Text::Display>
         <Hds::Text::Body @tag="p" @size="100" @color="faint" class="has-bottom-margin-m">
-          Connection credentials are sensitive information and the value cannot be read. Enable the input to update.
+          Connection credentials are sensitive information
+          {{#if @destination.isNew}} used to authenticate with the destination. {{else}}
+            and the value cannot be read. Enable the input to update.
+          {{/if}}
         </Hds::Text::Body>
-        {{#each fields as |attr|}}
-          <EnableInput data-test-enable-field={{attr.name}} class="field" @attr={{attr}}>
-            <FormField @attr={{attr}} @model={{@destination}} @modelValidations={{this.modelValidations}} />
-          </EnableInput>
-        {{/each}}
-      {{else}}
+      {{/if}}
+      {{#if (eq group "config")}}
+        <hr class="has-top-margin-xl has-bottom-margin-l has-background-gray-200" />
+        <Hds::Text::Display @tag="h2" @size="400" @weight="bold">Advanced configuration</Hds::Text::Display>
+        <Hds::Text::Body @tag="p" @size="100" @color="faint" class="has-bottom-margin-m">
+          Configuration options for the destination.
+        </Hds::Text::Body>
+      {{/if}}
+      {{! CREATE fields }}
+      {{#if @destination.isNew}}
```

Comment:
Instead of putting this `{{#if}}` here I wonder if we can simplify this logic and move this block to inside the `{{each}}`? That way we can combine it with the conditional below as well `{{#if (eq group "Credentials")}}`

```hbs
        {{#each fields as |attr|}}
          {{#if (and (eq group "Credentials") (not @destination.isNew))}}
            <EnableInput data-test-enable-field={{attr.name}} class="field" @attr={{attr}}>
              <FormField @attr={{attr}} @model={{@destination}} @modelValidations={{this.modelValidations}} />
            </EnableInput>
          {{else}}
            <FormField
              @attr={{attr}}
              @model={{@destination}}
              @modelValidations={{this.modelValidations}}
              @onKeyUp={{this.updateWarningValidation}}
            />
          {{/if}}
        {{/each}}